src/PartialCircuitBreaker.php

```diff
@@ -62,23 +67,37 @@ public function getState()
      */
     public function isOpened()
     {
-        return States::OPEN_STATE === $this->currentPlace->getState();
+        return State::OPEN_STATE === $this->currentPlace->getState();
     }
 
     /**
      * {@inheritdoc}
      */
     public function isHalfOpened()
     {
-        return States::HALF_OPEN_STATE === $this->currentPlace->getState();
+        return State::HALF_OPEN_STATE === $this->currentPlace->getState();
     }
 
     /**
      * {@inheritdoc}
      */
     public function isClosed()
     {
-        return States::CLOSED_STATE === $this->currentPlace->getState();
+        return State::CLOSED_STATE === $this->currentPlace->getState();
+    }
+
+    /**
+     * @param callable|null $fallback
+     *
+     * @return string
+     */
+    protected function callFallback(callable $fallback = null)
+    {
+        if (null === $fallback) {
+            return '';
```

Comment:
any reason to return an empty string. Could be maybe easier to return null or something different.
What if the callback needs to return '' 🤔

Reply:
I originally wanted to return `false` but then the return type would be multiple (`string|false`) without bringing much value.
I prefer to keep the single type return and therefore return empty string. But if one needs a special return value he can always define a `defaultFallback` or define a fallback per call. So as a default value I think it does the job.

Reply:
Because it's a fallback it can return anything no?
I think it should return void if there's no fallback. ping @matks

Reply:
the fallback is used to return something in the `call` method, so a string is expected
of course if you want you can return anything, but the default one should match the expected type